src/tests/solution.test.js

```diff
@@ -1,4 +1,4 @@
-import { renderData, prepareData } from "../src/solution";
+import { renderData, prepareData } from "../solution";
```

Comment:
We need to move or delete the original file.

Reply:
What to do you mean to erase the original file? 🤔

Reply:
Oh! nvmd! Just noticed we changed `solution.test.js` location. Its ok 👍🏽